mempool/src/pool.rs

```diff
@@ -204,6 +186,7 @@ pub struct Mempool<M: GetMemoryUsage + 'static + Send + std::marker::Sync> {
     clock: TimeGetter,
     memory_usage_estimator: M,
     events_controller: EventsController<MempoolEvent>,
+    receiver: mpsc::UnboundedReceiver<MempoolMethodCall>,
```

Comment:
I can't comment on the struct line, so I'll write it here.

Why should `lib.rs` have access to `struct Mempool`? This should be completely hidden inside somewhere. This is why I create the `detail` submodules often.

Reply:
Hid `Mempool` away by making `pool` a private submodule.